Backend/app/kafka-consumers/src/Environment.hs

```diff
@@ -95,7 +104,8 @@ data AppCfg = AppCfg
     httpClientOptions :: HttpClientOptions,
     enableRedisLatencyLogging :: Bool,
     enablePrometheusMetricLogging :: Bool,
-    tables :: Tables
+    tables :: Tables,
+    s3Config :: S3Config
```

Comment:
let have it as Maybe, and enforce it after consumerType check at runTime (i.e. make the app crash if the `consumerType` mandates s3Env but it's not given in dhall because of it being maybe in AppCfg) as not all consumers need s3 env

Reply:
Fixed